Lib/test/_test_multiprocessing.py

```diff
@@ -3736,6 +3736,23 @@ def test_shared_memory_across_processes(self):
 
         sms.close()
 
+    @unittest.skipIf(WIN32, "test not feasible in Windows")
```

Comment:
Wouldn't `os.name != 'posix'` be a safer condition? There may be other strange platforms besides Windows.

Reply:
Good idea.